web/src/main/java/com/navercorp/pinpoint/web/view/TransactionInfoViewModel.java

```diff
@@ -163,6 +163,10 @@ public String getDisableButtonMessage() {
                 }
                 first = false;
             }
+
+            if((record.getApiType() != "ASYNC") && (record.getElapsed() == 0)) {
```

Comment:
```suggestion
            if (!"ASYNC".equals(record.getApiType()) && (record.getElapsed() == 0)) {
```

Use the string.equals(),  not the == operator